app/scripts/constants.js

```diff
@@ -82,15 +82,21 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
   // when users navigate away from a page without saving.
   DISABLE_CONFIRM_ON_EXIT: false,
 
-  // This blacklist hides certain kinds from the "Other Resources" page because they are unpersisted, disallowed for most end users, or not supported by openshift but exist in kubernetes
+  // Disable the new landing page and service catalog experience.
```

Comment:
Just curious, how long will we keep this flag? Assuming we won't support the old landing page forever.

Reply:
I was thinking we'd remove 3.8